conda-forge.yml

```diff
@@ -12,3 +12,5 @@ provider:
 test_on_native_only: true
 conda_build:
   pkg_format: '2'
+azure:
+  store_build_artifacts: True
```

Comment:
FYI this is the source of the spurious failures on windows 

https://github.com/conda-forge/conda-smithy/pull/1724
https://github.com/conda-forge/conda-smithy/issues/1725